org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+    buildsystem: autotools
+    config-opts:
+      - --enable-display
+      - --enable-geometry
+      - --enable-graphics_ctx
+      - --enable-mediactrl
+      - --enable-sound
+      - --enable-webview
+      - --with-expat=sys
+      - --with-gtk=3
+      - --with-libjpeg=sys
+      - --with-libpng=sys
+      - --with-libtiff=sys
+      #- --with-opengl
+      - --with-zlib
+    sources:
+      - type: archive
+        url: 'https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2'
+        sha1: 406ac736f61d88a3a866aa501e01e408a642c6e7
+    cleanup:
+      - /bin
+      - /include
+      - /lib/wx
+      - /share
+
+  #-------------------------------------------------------------------------------
+  # wxC.
+  #
+  # TODO: use specific module version instead of the last commit
+  #-------------------------------------------------------------------------------
+  - name: wxc
+    buildsystem: simple
+    sources:
+      - type: git
+        url: 'https://gitlab.com/neva_blyad/wxc'
+    build-options:
+      env:
+        - ARCH=x86_64-linux-gnu # TODO: do cross-platform
+    build-commands:
+      - sed -i 's|prefix=/usr|prefix=/app|' wxc.pc.unix
+      - scons
+      - scons install     --prefix=/app/ --arch=$ARCH
+      - scons install-dev --prefix=/app/ --arch=$ARCH
+      - mkdir -p /app/etc/
+      - echo /app/lib/x86_64-linux-gnu/ > /app/etc/ld.so.conf
+    cleanup:
+      - /include
+      - /lib/x86_64-linux-gnu/libwxc.a
+      - /lib/x86_64-linux-gnu/libwxc.so
+      - /lib/x86_64-linux-gnu/pkgconfig
+
+  #-------------------------------------------------------------------------------
+  # libpeas
+  #-------------------------------------------------------------------------------
+  - name: libpeas
+    buildsystem: meson
+    sources:
+      - type: git
+        url: 'https://gitlab.gnome.org/GNOME/libpeas.git'
+        tag: libpeas-1.34.0
+    build-commands:
+      - ln -s /app/share/gir-1.0/Peas-1.0.gir /usr/share/gir-1.0/
+    cleanup:
+      - /bin
+      - /include
+      - /lib/debug
+      - /lib/peas-demo
+      - /lib/pkgconfig
+      - /share
+
+  #-------------------------------------------------------------------------------
+  # meson-perl.
+  #
+  # TODO: patch vs full source.
+  #-------------------------------------------------------------------------------
+  #- name: meson-perl
+  #  buildsystem: meson
+
+  #-------------------------------------------------------------------------------
+  # libpeas-perl.
+  #
+  # TODO
+  #-------------------------------------------------------------------------------
+  #- name: libpeas
+  #  buildsystem: meson
+  #  sources:
+  #    - type: git
+  #      url: 'https://gitlab.com/neva_blyad/libpeas-perl'
+  #      branch: main
+
+  #-------------------------------------------------------------------------------
+  # libgee
+  #-------------------------------------------------------------------------------
+  - name: libgee
```

Comment:
if you use the GNOME runtime it's in there.

Reply:
Sorry, no.
The app is not a GNOME app.